frameworks/CSharp/aspnet/benchmark_config.json

```diff
@@ -203,7 +203,7 @@
       "notes": "",
       "versus": ""
     },
-    "mono": {
+    "default": {
```

Comment:
These changes should not be necessary. We already have the "os" key to denote on which operating system the test can run. The fact that the default test does not run on linux is of no concern.